### PR TITLE
RubocopFixes

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -6,25 +6,7 @@
 # Note that changes in the inspected code, or installation of new
```
Comment (tphoney) [inline .rubocop_todo.yml]: This file should be empty. if everything is fixed

**Comment by tphoney** [comment]
Could you squash the commits. :)

**Comment by david22swan** [comment]
Have made the changes.

**Comment by tphoney** [comment]
@david22swan looks to be missing the todo file

**Comment by david22swan** [comment]
@tphoney Todo file already exists

**Comment by david22swan** [comment]
I'm an idiot, branched out at a point before todo file was filled.

**Comment by tphoney** [comment]
Great work @david22swan !!! 👍

**Pull request merged**